docs/conf.py

```diff
@@ -90,6 +90,12 @@
  "esm_version_checker",
 ]
 esm_tools_modules.remove("esm_tools")
+# Uncomment the following if you don't want to compile the docstrings
```

Comment:
Should these lines are commented out by default?

Reply:
This is a deprecated comment. Now it is easier to avoid compiling the docstrings. You just need to create a `.docstrings.yml` with the content `docstrings: 0`. I'll change this comment.

Reply:
Now I got it. Thanks.